gateway/api_loader.go

```diff
@@ -782,9 +778,6 @@ func loadApps(specs []*APISpec) {
 	shouldTrace := trace.IsEnabled()
 	for _, spec := range specs {
 		func() {
-			if strings.Contains(spec.Proxy.TargetURL, "h2c://") {
-				spec.Protocol = "h2c"
```

Comment:
h2c protocol was deprecated from the previous change (http protocol automatically handle it). And goal is that User->Tyk logic should not depend on Tyk->Upstream logic.